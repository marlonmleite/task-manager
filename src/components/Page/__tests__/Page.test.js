import React from 'react'
import { shallow } from 'enzyme'
import { Page, PageContent, PageFilter, PageTop, PageTitle } from '../'

describe('Page', () => {
  it('should render page', () => {
    const page = shallow(<Page />)

    expect(page).toMatchSnapshot()
  })

  it('should render page content', () => {
    const page = shallow(<PageContent />)

    expect(page).toMatchSnapshot()
  })

  it('should render page filter', () => {
    const page = shallow(<PageFilter />)

    expect(page).toMatchSnapshot()
  })

  it('should render page top', () => {
    const page = shallow(<PageTop />)

    expect(page).toMatchSnapshot()
  })

  it('should render page title', () => {
    const page = shallow(<PageTitle />)

    expect(page).toMatchSnapshot()
  })
})
