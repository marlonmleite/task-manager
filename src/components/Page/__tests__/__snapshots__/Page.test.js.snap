// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page should render page 1`] = `
.c0 {
  width: 100%;
  padding: 1.625rem 6.25rem 3.75rem 3.75rem;
  box-sizing: border-box;
  overflow-y: auto;
}

<div
  className="c0"
/>
`;

exports[`Page should render page content 1`] = `
.c0 {
  width: 100%;
  box-shadow: 0 0 6px 0 #00000040;
  box-sizing: border-box;
  background-color: #fff;
  margin-bottom: 30px;
}

<div
  className="c0"
/>
`;

exports[`Page should render page filter 1`] = `
.c0 {
  width: 100%;
}

<div
  className="c0"
/>
`;

exports[`Page should render page title 1`] = `
.c0 {
  color: #616161;
  text-transform: uppercase;
  margin: 0;
  padding: 0 15px;
  width: 100%;
}

<h5
  className="c0"
/>
`;

exports[`Page should render page top 1`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-bottom: 10px;
}

<div
  className="c0"
/>
`;
